package.json

```diff
@@ -48,6 +48,7 @@
     "prepare": "lerna run prepare",
     "test": "yarn eslint \"/**/*.{ts}\"",
     "unit-tests": "jest --maxWorkers=4",
+    "coverage": "npx jest --coverage",
```

Comment:
We're not using `npx` anywhere so I think we should stick with yarn to be consistent.